src/server/api/fileio.js

```diff
@@ -64,5 +64,13 @@ export default function fileIORouter() {
     logger.info("adding the GET /api/download handler");
     api.get("/download", auth.authnTokenMiddleware, downloadHandler);
 
+    logger.info("adding the GET /api/downloadText handler");
+    api.get("/downloadText", function (req, res) {
+        res.set({
+            "Content-Disposition": 'attachment; filename="de-untitled.txt"',
```

Comment:
If I'm understanding right, the `filename` property is a default name that gets auto-filled in for the `Save As` dialog that pops up for the user?  Maybe `cyverse-download.txt` or something similar might be a default that users in a hurry won't have to change too frequently?

Reply:
This logic is not used in `Save As`. This used when users request downloading of `links` to the local machine. I could name it to something like `de-links.txt` but this function is pretty generic so  I don't want to do that. May be I can a filename in query param ? 🤔

Reply:
You're right - for some reason I thought when you download something, you have an opportunity to change the name of the downloaded file, but I guess not.  It would be nice if the downloaded filename could be the same as the original file name 👍

Reply:
Some correction - This is not a downloading a file from Datastore. But rather, we create a new file in memory in Express with all the links requested by the user and then that file is downloaded.

Reply:
`Copy Link(s)` - Will copy links to clipboard
`Download` - Will download the links as a file to users local computer
`Save to file` - Will save to links to a file in the Data store.

Reply:
Got it - really my only hesitation was with the `untitled` part of the file name. If `de-links` is too specific then maybe `de-download` or something similar could work?

Reply:
I've been vaguely following this thread, and my feeling is: a query parameter is good if we want to have a suggested filename, but the default should just be `Content-Disposition: attachment` -- that will still set it to be downloaded, but without a filename specified. I don't think "de-untitled.txt" will help anyone, so it's better left off so the browser/client/user can decide what to name the file.

Reply:
Query Params seems to be best options but if you don't set a default file name, chrome downloads it as `downloadText.html`(HTML file) and firefox opens it as `downloadText`. I still think setting a default file name is good idea and that file name should start with `de`.

Reply:
Ah, that makes sense. It's unfortunate that the browser defaults aren't nicer, and `de-download.txt` does seem like a better option than theirs.